### PR TITLE
Stop queue config table cache update thread when the table is gone.

### PR DESCRIPTION
**Pull request by chtyim**
- Since we have one config table per namespace, the cache update thread should stop if the table doesn’t exists.

### CODE REVIEW

**Comment by chtyim** [comment]
Build pass: http://builds.cask.co/browse/CDAP-DUT906-1

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/coprocessor/ConsumerConfigCache.java [hunk 1]
```diff
@@ -161,18 +169,27 @@ public synchronized void updateCache() {
...
           updateConfig();
           long now = System.currentTimeMillis();
           if (now > (lastUpdated + configCacheUpdateFrequency)) {
-            updateCache();
+            try {
+              updateCache();
+              tableNotFoundCount = 0;
+            } catch (TableNotFoundException e) {
+              tableNotFoundCount++;
+              LOG.error("Queue config table not found: {}. Retries remaining {}",
```
Comment (jwang47) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/coprocessor/ConsumerConfigCache.java]: Maybe not log single retry failure in ERROR, but instead at INFO and log as ERROR when we stop trying?

**Comment by jwang47** [comment]
One minor comment, otherwise LGTM

**Comment by chtyim** [comment]
@jwang47 Updated the PR. Logic changed a bit, please take a look again.

**Comment by jwang47** [comment]
LGTM

**Comment by chtyim** [comment]
build passed after the latest change http://builds.cask.co/browse/CDAP-DUT906-3

**Pull request merged**